### PR TITLE
Crawltask exitcodes

### PR DESCRIPTION
**Pull request by brittainhard**

### CODE REVIEW

**Comment by brittainhard** [comment]
Ready for Review

File: source/apps/crawl_space/crawl_runners.py [hunk 1]
```diff
@@ -1,258 +0,0 @@
-"""Provides CrawlRunner subclasses to wrap crawl process execution and
```
Comment (ahmadia) [inline source/apps/crawl_space/crawl_runners.py]: is this file redundant?

File: source/apps/crawl_space/crawl_supervisor.py [hunk 2]
```diff
@@ -1,98 +0,0 @@
-"""Crawl Supervisor is a free-standing script that monitors and logs output
```
Comment (ahmadia) [inline source/apps/crawl_space/crawl_supervisor.py]: is this now redundant as well?

File: source/apps/crawl_space/crawl_runners.py (hunk 1 above)
Comment (ahmadia) [inline source/apps/crawl_space/crawl_runners.py]: I see, noted in your commit log.

**Comment by brittainhard** [comment]
Merging if no comment.

File: source/apps/crawl_space/crawl_supervisor.py (hunk 2 above)
Comment (brittainhard) [inline source/apps/crawl_space/crawl_supervisor.py]: Yes, these files were the files which were previously running the crawls. Just took them out as a side thought.

**Pull request merged**